pkg/api/app.go

```diff
@@ -69,6 +73,7 @@ func newApp(apiKey string, scheduler SchedulerEmits, services services.Services,
 		return nil, err
 	}
 
+	settings.GenerationPeriod = cfg.GenerationPeriod
```

Comment:
Current argument name collides with the package name. Suggest to rename variable.